tcmur_cmd_handler.c

```diff
@@ -2267,6 +2279,22 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 		break;
 	}
```

Comment:
We need to move this down to the lib level code or move it up the code in tcmulib_command_complete to tcmur_command_complete so they are the same level for when we do the lib split.

The other calls like tcmu_init/cleanup_timer_base then should stay in runner or move to the lib.

Either the lib should do it for the lib user (tcmu-runner) or the lib does not know about it and tcmu-runner does it all.